### ISSUE TITLE
Group together "Decimal", "Float" and "Integer" in the field type drop-down menu.

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#504]
This one is the same as https://www.drupal.org/node/2072533 (already committed in D8)

This list is currently like so:
- Boolean
- **Decimal**
- File
- **Float**
- Image
- **Integer**
- List (float)
- List (integer)
- List (text)
- Long text
- Long text with summary
- Term reference
- Text

In D8 it was changed to
- Boolean
- File
- Image
- List (float)
- List (integer)
- List (text)
- **Number (decimal)**
- **Number (float)**
- **Number (integer)**
- Long text
- Long text with summary
- Term reference
- Text

In other words, all variants of numbers is grouped together same as variant of lists are.
